lib/Service/OpenProjectAPIService.php

```diff
@@ -35,6 +35,7 @@
 use OCA\TermsOfService\Db\Mapper\TermsMapper;
 use OCA\UserOIDC\Db\ProviderMapper;
 use OCA\UserOIDC\Exception\TokenExchangeFailedException;
+use OCA\UserOIDC\User\Backend;
```

Comment:
maybe make it more explicit what backend it is about
```suggestion
use OCA\UserOIDC\User\Backend as OIDCBackend;
```